.clj-kondo/config.edn

```diff
@@ -10,7 +10,6 @@
   :missing-docstring            {:level :warning}
   :missing-else-branch          {:level :warning}
   :namespace-name-mismatch      {:level :warning}
-  :non-arg-vec-return-type-hint {:level :warning}
```

Comment:
you can probably revert all the changes in this file now

Reply:
yeah thanks baf86f7da3824555cb66ac948295415f811cd914